harness/determined/cli/_util.py

```diff
@@ -85,28 +84,27 @@ def make_pagination_args(
 default_pagination_args = make_pagination_args()
 
 
-def login_sdk_client(func: Callable[[argparse.Namespace], Any]) -> Callable[..., Any]:
-    @functools.wraps(func)
-    def f(namespace: argparse.Namespace) -> Any:
-        client.login(master=namespace.master, user=namespace.user)
-        return func(namespace)
-
-    return f
+def unauth_session(args: argparse.Namespace) -> api.UnauthSession:
+    master_url = args.master or util.get_default_master_address()
+    return api.UnauthSession(master=master_url, cert=cli.cert)
 
 
 def setup_session(args: argparse.Namespace) -> api.Session:
     master_url = args.master or util.get_default_master_address()
-    cert = certs.default_load(master_url)
-    retry = api.default_retry()
-
-    return api.Session(master_url, args.user, authentication.cli_auth, cert, retry)
+    utp = authentication.login_with_cache(
```

Comment:
I like how this refactor has landed for the CLI.  I'm not asking for anything from this PR, in this comment.

I think this is a place where this refactor is a potential stepping-stone to a later state where we've set up a login path that is only for this particular use case. Curious if you think the same and can express it in a sentence or two for the wiki.

Reply:
I'm not sure I understand what you mean